src/main/resources/assets/wurst/lang/ja_jp.json

```diff
@@ -165,5 +169,15 @@
   "gui.wurst.altmanager.folder_error.title": "フォルダー「.Wurst encryption」の作成に失敗しました。",
   "gui.wurst.altmanager.folder_error.message": "Wurstのフォルダーへのアクセス権限を拒否してしまっている可能性があります。\nアクセス権限がないと、AltManagerはAltリストを暗号化および複合化することができません。\n引き続きAltManagerを使用することはできますが、新たに作成したAltは保存されません。\n\nエラーの全容:\n%s",
   "gui.wurst.altmanager.empty.title": "Altリストは空です。",
-  "gui.wurst.altmanager.empty.message": "手始めに、ランダムなAltをいくつか追加しますか？"
+  "gui.wurst.altmanager.empty.message": "手始めに、ランダムなAltをいくつか追加しますか？",
+  "gui.wurst.nochatreports.unsafe_server.title": "§4§l警告:§r 安全でないサーバー",
+  "gui.wurst.nochatreports.unsafe_server.message": "このサーバーでは、チャット署名を有効にする必要があります, そのため、アカウントが不正なチャットレポートの危険にさらされます。\n\nチャット署名のブロックを解除してから再接続すると、このサーバーに参加できます。 その場合は、チャットをまったく使用しないことを検討してください。 または、BANされても大丈夫な代替アカウントでプレイします。\n\nこれがあなたのサーバーである場合、server.properties で'enforce-secure-profile'を false に設定することでこれを修正できます。典型的なモジャンのやり方で, この設定は、見た目とは逆です。",
+  "toast.wurst.nochatreports.unsafe_server.title": "チャットメッセージを報告できます",
+  "toast.wurst.nochatreports.unsafe_server.message": "このサーバーで送信されたメッセージはMojangに報告される可能性があり、アカウントがグローバルBANされる可能性があります",
+  "gui.wurst.nochatreports.ncr_mod_server.title": "\"チャットレポートなし\" サーバー",
+  "gui.wurst.nochatreports.ncr_mod_server.message": "このサーバーは \"チャットレポートなし\" mod を使用しています。\n\nチャット署名をブロックし、VanillaSpoof を無効にすると、このサーバーに参加できます。",

```

Comment:
I think "No Chat Reports" should be kept in English, since that's the official name of the mod.

Reply:
I fixed it.
is it ok?